Makefile

```diff
@@ -99,6 +99,18 @@ deploy-eth: compile-eth
 	npx hardhat run --network $(NETWORK) scripts/1_deploy_gnosis_safe.js && \
 	npx hardhat run --network $(NETWORK) scripts/2_deploy_bridge_contracts.js
 
+.PHONY: deploy-eth-live
```

Comment:
Would be nice to add one make instruction that just takes mnemonics (or they could be input to configs before deployment, both are OK) and deploys all contracts on testnets - now it's not immediately clear how to do it.